Dockerfile.test

```
@@ -0,0 +1,26 @@
+# Use the official PostgreSQL 16.1 image as the base image
+ARG PG_VERSION=16.1
+FROM postgres:${PG_VERSION}-alpine
+
+ARG PGTAP_VERSION=v1.3.2
+ENV DOCKERIZE_VERSION=v0.6.1
+
+RUN apk add --no-cache --update curl wget git openssl build-base make perl perl-dev \
+    && rm -rf /var/cache/apk/* /tmp/* \
+    && cpan TAP::Parser::SourceHandler::pgTAP \
+    && git clone https://github.com/theory/pgtap.git \
+    && cd pgtap \
+    && git checkout tags/$PGTAP_VERSION \
+    && make
+
+WORKDIR /
+
+COPY test/unit/*.sql /test/
+COPY db/seed/*.sql /seed/
+COPY test/test.sh /test/
+COPY run-pg-and-pgtap-test.sh /test/run-pg-and-pgtap-test.sh
```

Could you please try the following? Always two image layers less 😊 
```suggestion
COPY run-pg-and-pgtap-test.sh test/test.sh test/unit/*.sql /test/
COPY db/seed/*.sql /seed/
```